Rocket.Chat/fr.lproj/Localizable.strings

```diff
@@ -0,0 +1,126 @@
+/*
+ Localizable.strings
+ Rocket.Chat
+ 
+ Created by Rafael K. Streit on 8/27/16.
+ Copyright © 2016 Rocket.Chat. All rights reserved.
+ */
+
+
+// Global
+"global.ok" = "OK";
+"global.cancel" = "Annuler";
+
+// Errors
+"error.socket.default_error_title" = "Oups!";
+"error.socket.default_error_message" = "Une erreur s'est produite, veuillez réessayer.";
+
+// Alerts
+"alert.connection.invalid_url.title" = "Oups!";
+"alert.connection.invalid_url.message" = "L'URL saisie est invalide. Veuillez la vérifier et réessayer, s'il vous plaît!";
+
+"alert.connection.invalid_version.title" = "Oups!";
+"alert.connection.invalid_version.message" = "Le serveur auquel vous essayez de vous connecter utilise une version qui n'est plus supportée par l'app: %@.\n\nLa version %@ est requise.";
+
+"alert.settings.set_mail_app.title" = "Oups!";
+"alert.settings.set_mail_app.message" = "Aucun compte mail. Veuillez configurer un compte mail afin de pouvoir envoyer des emails.";
+
+"alert.feature.wip.title" = "Oups!";
+"alert.feature.wip.message" = "Cette fonctionnalité est en cours de développement, restez au courant!";
+
```

Comment:
Could you please, add this new strings to the file (we have then in our English, if you want to look into it)?

```
"alert.socket_error.invalid_user.title" = "Session expired";
"alert.socket_error.invalid_user.message" = "Your session has expired. Please log in again.";

"alert.login_service_error.title" = "Authentication Error";
"alert.login_service_error.message" = "Could not authenticate using this service. Please, try again.";
```